selectors.yml

```diff
@@ -373,4 +373,16 @@ selectors:
     name: "private-testnet-mica"
   48900:
     selector: 17198166215261833993
-    name: "ethereum-mainnet-zircuit-1"
\ No newline at end of file
+    name: "ethereum-mainnet-zircuit-1"
+  223:
+    selector: 5406759801798337480
+    name: "bitcoin-mainnet-bsquared-1"
+  200901:
+    selector: 7937294810946806131
+    name: "bitcoin-mainnet-bitlayer-1"
+  60808:
+    selector: 3849287863852499584
+    name: "ethereum-mainnet-bob-1"
```

Comment:
Is Bob anchored to ethereum-mainnet?
From what I can read in their site, they "derive security from Bitcoin", so it should be considered a rollup on top of `bitcoin-mainnet`?

Reply:
I wrongly assumed it was in both bitcoing and ethereum: "BOB ("Build on Bitcoin") is a first-of-its-kind hybrid Layer-2 powered by Bitcoin and Ethereum. " https://docs.gobob.xyz/docs/learn/introduction/stack

But from here "We see BOB as a collective undertaking to scale Bitcoin the right way: inheriting security from Bitcoin while providing usable decentralization for builders today without waiting for hard forks." https://docs.gobob.xyz/docs/drafts/vision it seems you are right.

will change `bitcoin-mainnet-bob-1` to `bitcoin-mainnet-bob-1` and also `ethereum-testnet-sepolia-bob-1` to `bitcoin-testnet-sepolia-bob-1`, thanks @andrevmatos